catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -511,6 +530,7 @@ public void destroy() {
     config.put("basicSearchTemporalSelectionDefault", basicSearchTemporalSelectionDefault);
     config.put("basicSearchMatchType", basicSearchMatchType);
     config.put("useHyphensInUuid", uuidGenerator.useHyphens());
+    config.put("keywords", keywords);
```

Comment:
❓  Would "i18n" be clearer than "keywords"?

Reply:
I'm not sure to be honest. `keywords` is more ambiguous than `i18n` so I'll change it. Keywords came from when this was called `keyword-registry` so it made more sense back then

Reply:
I'm not sure which is better to be honest. Keywords describes the type of the data better but i18n is more specific as to its purpose.

Reply:
I am thinking in the future when all labels and messages support i18n.  They will often be phrases and sentences which seems different from keywords.

Reply:
That works for me. Done.